File: packages/ios-client/Sources/MembraneRTC/Types/AuthError.swift
public enum AuthError: String, CaseIterable {
    case missing_token = "missing token"
    case invalid_token = "invalid token"
    case expired_token = "expired token"
    case room_not_found = "room not found"
    case peer_not_found = "peer not found"
    case peer_already_connected = "peer already connected"
}

File: packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/models/AuthError.kt
package com.fishjamcloud.client.models

enum class AuthError(
  val error: String
) {
  MISSING_TOKEN("missing token"),
  INVALID_TOKEN("invalid token"),
  EXPIRED_TOKEN("expired token"),
  ROOM_NOT_FOUND("room not found"),
  PEER_NOT_FOUND("peer not found"),
  PEER_CONNECTED("peer already connected");

  companion object {
    fun isAuthError(error: String): Boolean = values().firstOrNull { v -> v.error == error } != null

    fun fromString(error: String): AuthError = values().first { v -> v.error == error }
  }
}
